pinot-common/src/main/java/com/linkedin/pinot/pql/parsers/Pql2AstListener.java

```diff
@@ -53,6 +55,11 @@
 public class Pql2AstListener extends PQL2BaseListener {
   Stack<AstNode> _nodeStack = new Stack<>();
   AstNode _rootNode = null;
+  private String _expression;
```

Comment:
Since it can be either (we can compile fragments of the query), I kept is as expression.